### ISSUE TITLE
Badge Ideas

### ISSUE THREAD

**Issue by GuvBubbs** [madarasz/always-be-running#20]
Badges should both be a point of pride and encourage people to use the site in the way we want.

There might come a point when people have a lot of badges so you may want to add an option for uses to decide what badges to display.

Not sure what to call things but I'll just make a list of ideas. Some of these will be for features yet to come.

Completed profile - (Set display name, timezone and wrote a bio)
Attached photos to an event
Attached Videos to an event
Featured in a video on an event
Have an event with all places claimed
Commented on X events you were in.
Commented on X events you weren't in.
Register for x events.
Goes to weekly events in home city (I don't see people registering for these otherwise)
Claimed IDs for every runner faction
Claimed IDs for every corp faction
Took a mini faction to a Tournament 
Decklist of the week (Not sure if you can pull this data but would be cool)
Reviewed X cards on NRDB (Not sure if you can pull this data but would be cool)

**Comment by GuvBubbs** [comment]
Claimed a decklist for someone else.

**Comment by madarasz** [comment]
Badges are listed here:
http://www.alwaysberunning.net/badges

Added the every runner/corp and mini-faction ideas as badges.

**Comment by GuvBubbs** [comment]
- [ ] User has donated NRDB & ABR

**Comment by GuvBubbs** [comment]
- [ ] Road to Worlds - Went to Store Champs, Regionals, Nationals & Worlds in one year.

**Comment by madarasz** [comment]
I added the badges based on NetrunnerDB reputation. This sort of substitutes your ideas on decklist of the week and card reviewing.

I think I will add a "Road to Worlds" badge without the Worlds participation, so users can get it without traveling to the the U.S.

**Comment by GuvBubbs** [comment]
Yeah at first I thought it should be "Road to Nationals" But i'm going to worlds next year sooooo ha ha

**Comment by GuvBubbs** [comment]
If you cut world then I would add a GNK

**Comment by GuvBubbs** [comment]
Are you able to make it within the same year?

**Comment by madarasz** [comment]
The problem with that is 2017 Store Champs already started in late 2016. It would just complicate things too much. I know it would be very thematic, but I would drop that requirement. It is easier to get the badge, you just have to participate in Store, Regionals and Regionals. It is still a progression, "road" nevertheless.

**Comment by GuvBubbs** [comment]
You mean in order?

**Comment by madarasz** [comment]
I'm keeping it simple. Do not over-complicate things, let it go :) The order does not matter, the location does not matter, just that the user has participated at least in one Store, Regionals and Nationals ever. (GNKs are not part of the FFG "championship" ladder, they give no trophy/plaque or buys). I'm keeping the Worlds participation off the requirement, because it's not trivial to travel to the US.

**Comment by GuvBubbs** [comment]
Okay okay okay =D

**Comment by madarasz** [comment]
Added notification for new badges. (At this very moment all badges will count as unseen new badges.) Hopefully users will notice this badge business and their profile page more.

**Comment by GuvBubbs** [comment]
Oh that's a great idea. 

What are you thoughts on the Completed profile badge?

And the ability to pick what's on display?

**Comment by madarasz** [comment]
I'm holding the completed profile badge off for a while. I want to add some more fields (eg. country of user, favorite runner/corp faction) and than have that badge.

'Claimed decklist for someone else' is tricky especially that we already have some claims. Rather not do that.

Just added 'Champion of Sorts' for winning a something that is not Store/Regionals/Nationals/Worlds.

**Comment by GuvBubbs** [comment]
Just a thought. Might be worth adding a Road to Worlds equivalent for TO's.
When you run the events to don't get to place/claim.

- [ ] Community Hero - TO'ed Store, Regionals & Nationals

**Comment by madarasz** [comment]
Great idea, will add tomorrow! Also wanting to add:
- [ ] Traveller - Claimed spots in 3 different countries hosting a tournament

**Comment by GuvBubbs** [comment]
Ahh I was thinking about something like that but then you might then need TO vetting...

**Comment by madarasz** [comment]
Well, well, well... Look who's Community Champion :D

- [ ] Virtual Persona - claimed in 3 online tournaments
(will add later, we still don't have any online tournaments)

Let's put these badge ideas to rest for a while.

**Comment by madarasz** [comment]
Just couldn't resist and added a charity badge, we already have 2 great charities. Unfortunately there's no way to automatically give these out.

**Comment by GuvBubbs** [comment]
Yeah good call. There are bigger things to tackle.

**Comment by GuvBubbs** [comment]
I know this will mean I'll get another badge... But maybe a badge for having a featured event.

**Comment by GuvBubbs** [comment]
Just looking at my badge ideas...

It should be cool if you could if you could tag users in videos. Then you could have this badge

_**Featured in a video on an event**_

**Comment by madarasz** [comment]
I added the featured event badge.

Could you open a feature request for tagging users in videos?
